atlante/atlante.go

```diff
@@ -33,9 +33,12 @@ type ImgStruct struct {
 	intermediate bool
 	// Have we generated the file?
 	// This will allow us to generate the file only when requested
-	generated bool
-	lck       sync.Mutex
-	image     *image.Image
+	generated                 bool
+	lck                       sync.Mutex
+	image                     *image.Image
```

Comment:
image.Image is an interface. What do we benefit by making it a pointer here?

Reply:
image is not the standard go library image here. It's the mbgl/image library. Image is a concrete type.  We talked about this before. :P